Cubical/Data/Nat/Primes/Base.agda

```diff
@@ -0,0 +1,25 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Data.Nat.Primes.Base where
+
+open import Cubical.Data.Nat.Primes.Imports
```

Comment:
I thinks it is okay to have an `Imports` file, but in the case of `Base` it should be avoided to use something like that, since that could get involved in solving problems with cyclic dependencies later on.